library/src/main/java/com/chuckerteam/chucker/api/ChuckerInterceptor.kt

```diff
@@ -28,8 +28,8 @@ private const val MAX_BLOB_SIZE = 1000_000L
  * @param maxContentLength The maximum length for request and response content
  * before they are truncated. Warning: setting this value too high may cause unexpected
  * results.
- * @param headersToRedact List of headers that you want to redact. They will be not be shown in
- * the ChuckerUI but will be replaced with a `**`.
+ * @param headersToRedact Set of headers that you want to redact. They will be replaced
```

Comment:
`Set` is probably not the best word here. Ideally I would replace it with either:

- `Array` of headers... (as the `vararg` is compiled into a `String[]`)
- One or more headers...

Reply:
Probably, I could use [Set] there, like I did in  #190, so it won't confuse anyone. What do you think?

Reply:
Yeah that makes sense 👍

Reply:
Deal. Will do in the morning.